cli/deploy-moe-autoscale.sh

```diff
@@ -21,7 +21,7 @@ DEPLOYMENT_RESOURCE_ID=$(az ml online-deployment show -e $ENDPOINT_NAME -n $DEPL
 # ARM id of the deployment. todo: change to --query "id"
 ENDPOINT_RESOURCE_ID=$(az ml online-endpoint show -n $ENDPOINT_NAME -o tsv --query "properties.\"azureml.onlineendpointid\"")
 # set a unique name for autoscale settings for this deployment. The below will append a random number to make the name unique.
-AUTOSCALE_SETTINGS_NAME=autoscale-`echo $RANDOM`
+AUTOSCALE_SETTINGS_NAME=autoscale-$ENDPOINT_NAME-$DEPLOYMENT_NAME-`echo $RANDOM`
```

Comment:
RANDOM will generate IDs from 0 to 32767. I recommend instead just computing a hash of string [WorkspaceName + EndpointName + DeploymentName] and then appending to the same string.

Reply:
thanks for the feedback prior the PR as well - helpful! For the samples i'm going to keep it simple.

Reply:
If you have workspace name available, prepend it to your string as well to minimize collisions.

Reply:
noted.